### DISCUSSION TITLE
Proposal to Merge MOA Project Changes into searxng

### DISCUSSION THREAD

**Discussion by mehr32** [searxng/searxng#3736] [General]
Hello, I am the maintainer of the repository [MOA](https://github.com/moa-engine/MOA). For various reasons, we are unable to continue developing the project. Therefore, we have decided to submit the useful changes from our project to searxng, so at least our efforts won’t go to waste.  
I will gradually send smaller changes and search engines as PRs.  
However, I have a question regarding the format. (The MOA format is a fork of the simple searxng format.) There are two options for submission:  
1. We could turn it into a third-party fork of the searxng format, which we would not be able to maintain.  
2. We could merge the changes into the searxng format.  
What do you think?

**Comment by return42** [comment]
> The MOA format is a fork of the simple searxng format.)

If you say "simple searxng format" do you mean ["simple theme"](https://github.com/searxng/searxng/tree/master/searx/static/themes/simple)? If so; we don't have the resources to maintain an additional theme / whatever makes sense should be merged in the simple theme.

**Reply by mehr32** [reply]
So we will submit it as a PR for merging.
